data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/events/GetEvents.java

```diff
@@ -76,6 +72,20 @@ public GetEvents(
       final AsyncRunner asyncRunner,
       final TimeProvider timeProvider,
       final int maxPendingEvents) {
+    super(
+        EndpointMetadata.get(ROUTE)
+            .operationId("getEvents")
+            .summary("Subscribe to node events")
+            .description(
+                "Provides endpoint to subscribe to beacon node Server-Sent-Events stream. Consumers should use"
+                    + " [eventsource](https://html.spec.whatwg.org/multipage/server-sent-events.html#the-eventsource-interface)"
+                    + " implementation to listen on those events.\n\n"
+                    + "Servers _may_ send SSE comments beginning with `:` for any purpose, including to keep the"
+                    + " event stream connection alive in the presence of proxy servers.")
+            .tags(TAG_EVENTS, TAG_VALIDATOR_REQUIRED)
+            .queryParam(TOPICS_PARAMETER)
```

Comment:
Does this still work if you specify `/events?topics=block&topics=attestation` (ie specifying the arg multiple times instead of using comma separated values)?

I think it will and the open api looks ok to me but worth testing just to be sure.

Reply:
Yes seems to work!